org.scala-ide.sdt.core/src/org/scalaide/core/internal/builder/zinc/SbtInputs.scala

```diff
@@ -36,7 +36,8 @@ import org.scalaide.core.internal.ScalaPlugin
 import org.eclipse.core.resources.IContainer
 import org.eclipse.core.runtime.IPath
 
-/** Inputs-like class, but not implementing xsbti.compile.Inputs.
+/**
+ * Inputs-like class, but not implementing xsbti.compile.Inputs.
```

Comment:
Please review your formatting options. These partly doc changes shouldn't happen. Either let them as they are or format them correctly, i.e. one space indent for every line, starting on the second line.